query/src/storages/fuse/operations/read.rs

```diff
@@ -63,7 +63,6 @@ impl FuseTable {
 
         let part_stream = futures::stream::iter(iter);
 
-        let read_buffer_size = ctx.get_settings().get_storage_read_buffer_size()?;
```

Comment:
Does ctx.get_storage_read_buffer_size no used anymore?

Reply:
yes, still in use